modules/core/shared/src/main/scala/gem/config/StaticConfig.scala

```diff
@@ -47,35 +48,38 @@ object StaticConfig {
 
   import GmosConfig._
 
-  final case class GmosNorth(
+  @Lenses final case class GmosNorth(
     common:    GmosCommonStaticConfig,
     stageMode: GmosNorthStageMode
   ) extends StaticConfig.Impl(Instrument.GmosN)
 
+  @SuppressWarnings(Array("org.wartremover.warts.PublicInference"))
   object GmosNorth extends GmosNorthLenses {
+
     val Default: GmosNorth =
       GmosNorth(
         GmosCommonStaticConfig.Default,
         GmosNorthStageMode.FollowXy
       )
+
   }
 
-  trait GmosNorthLenses {
-    val Common: GmosNorth @> GmosCommonStaticConfig =
-      Lens((a, b) => a.copy(common = b), _.common)
+  trait GmosNorthLenses { this: GmosNorth.type =>
+
+    lazy val customRois: Lens[GmosNorth, Set[GmosCustomRoiEntry]] =
+      common ^|-> GmosCommonStaticConfig.customRois
```

Comment:
`a >=> b` becomes `a ^|-> b`.

Reply:
No big deal but I think I like >=> more.

Reply:
I'm used to the `>=>` for composing Monadic functions with a pointfree style ([Kleisli composition](https://hackage.haskell.org/package/base-4.10.0.0/docs/Control-Monad.html#v:-62--61--62-)). If `^|->` is an entire different thing, I think it's worth the rename, otherwise I'm not so sure.

Reply:
I propose the name "wind vane" for referring to `^|->`.

Reply:
![image](https://user-images.githubusercontent.com/215438/29876002-f7bfbc30-8d71-11e7-95df-4e0e84f13880.png)

Sorry, I couldn't help it.